google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -269,6 +278,11 @@ public static QueryParameterValue dateTime(String value) {
           return value.toString();
         }
         break;
+      case NUMERIC:
+        if (value instanceof BigDecimal) {
+          return value.toString();
```

Comment:
Sorry I just got to this. Do we need to round this value so that it only has 9 decimal points?

Reply:
In the Python code (https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5331), we just convert whatever value is provided into a string, since users have control over the precision of the values. If there are two may digits after the decimal point, BigQuery will return an error when it receives the value, so users are given an explicit signal that they should round or truncate rather than having it happen silently.

Reply:
+1 to explicit errors as opposed to silently dropping data.

Reply:
LGTM